integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -39,6 +43,22 @@ public Hooks(TestContext context) {
         this.context = context;
     }
 
+    @BeforeAll
```

Comment:
Please add `order` parameter to assure that the method is executed as first

Reply:
Sure